receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -232,16 +250,42 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 			continue
 		}
 
+		// If the metric name is equal to target_info, we use its labels as attributes of the resource
+		// Ref: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#resource-attributes-1
+		if ls.Has(labels.MetricName) && ls.Get(labels.MetricName) == "target_info" {
```

Comment:
```suggestion
		if ls.Get(labels.MetricName) == "target_info" {
```

We're doing this validation a few lines above already

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/05a8845ee34f26e73e18a8e7b40cc7012b798355